homeassistant/components/zha/climate.py

```diff
@@ -560,6 +561,61 @@ async def async_preset_handler_away(self, is_away: bool = False) -> bool:
         return res
 
 
+@STRICT_MATCH(
+    channel_names={CHANNEL_THERMOSTAT}, manufacturers="Eurotronic",
+)
+class EurotronicSpiritZigbeeThermostat(Thermostat):
+    """Eurotronic Spirit Zigbee Thermostat."""
+
+    manufacturer = 0x1037
+    HOST_FLAGS_ATTR = 0x4008
+    BOOST_FLAG = 0b00000100
+    OFF_MODE_FLAG = (
+        0b00010000  # if this flag is set, off mode enabled. set also disables.
+    )
+
+    def __init__(self, unique_id, zha_device, channels, **kwargs):
+        """Initialize ZHA Thermostat instance."""
+        super().__init__(unique_id, zha_device, channels, **kwargs)
+        self._presets = [PRESET_BOOST, PRESET_NONE]
+        self._supported_flags |= SUPPORT_PRESET_MODE
+
+    async def async_added_to_hass(self):
+        """Run when about to be added to Hass."""
+        await super().async_added_to_hass()
+
+    async def async_preset_handler_boost(self, boost: bool = False) -> bool:
+        """Set boost mode."""
+
+        mfg_code = self._zha_device.manufacturer_code
+        host_flags_raw, err = await self._thrm._cluster.read_attributes(
+            ["host_flags"], manufacturer=mfg_code
+        )
+
+        if host_flags_raw is not None and "host_flags" in host_flags_raw:
+            host_flags = host_flags_raw["host_flags"]
+        else:
+            host_flags = 1
```

Comment:
```suggestion
        host_flags = host_flags_raw.get("host_flags",  1)
```
Would this work? `host_flags_raw` can't be `None`